chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -31,6 +31,16 @@
 end
 
 is_rgw = node.roles.include?("ceph-radosgw")
+rgw_port = rgw_pemfile = nil
+if is_rgw
+  rgw_use_ssl = node["ceph"]["radosgw"]["ssl"]["enabled"]
+  if node["ceph"]["ha"]["radosgw"]["enabled"]
+    rgw_port = rgw_use_ssl ? node["ceph"]["ha"]["ports"]["radosgw_ssl"] : node["ceph"]["ha"]["ports"]["radosgw_plain"]
```

Comment:
Metrics/LineLength: Line is too long. [118/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)